Sources/TuistLoader/Loaders/ManifestLoader.swift

```diff
@@ -227,18 +225,12 @@ public class ManifestLoader: ManifestLoading {
             at: path
         ) + ["--tuist-dump"]
 
-        let result = System.shared
-            .observable(arguments, verbose: false, environment: environment.manifestLoadingVariables)
-            .toBlocking()
-            .materialize()
+        do {
+            let string = try System.shared.capture(arguments, verbose: false, environment: environment.manifestLoadingVariables)
 
-        switch result {
-        case let .completed(elements):
-            let output = elements.filter(\.isStandardOutput).map(\.value).reduce(into: Data()) { $0.append($1) }
-            guard let string = String(data: output, encoding: .utf8) else { return output }
-
-            guard let startTokenRange = string.range(of: ManifestLoader.startManifestToken) else { return output }
-            guard let endTokenRange = string.range(of: ManifestLoader.endManifestToken) else { return output }
+            guard let startTokenRange = string.range(of: ManifestLoader.startManifestToken)
+            else { return string.data(using: .utf8)! }
```

Comment:
For consistency with the line below
```suggestion
            guard let startTokenRange = string.range(of: ManifestLoader.startManifestToken) else { return string.data(using: .utf8)! }
```

Reply:
That's what I did initially, but SwiftFormat complained:

```
ManifestLoader.swift:231:1: warning: (wrap) Wrap lines that exceed the specified maximum width.
```

If you want, I can instead change the other line to match.

Reply:
Sorry, fixed it 🙏

Reply:
Works for me.